dev/create-release/create-release.sh

```diff
@@ -228,8 +228,8 @@ if [[ ! "$@" =~ --skip-package ]]; then
 
   # We increment the Zinc port each time to avoid OOM's and other craziness if multiple builds
   # share the same Zinc server.
-  make_binary_release "hadoop1" "-Phive -Phive-thriftserver -Dhadoop.version=1.0.4" "3030" &
-  make_binary_release "hadoop1-scala2.11" "-Phive -Dscala-2.11" "3031" &
+  make_binary_release "hadoop2" "-Phive -Phive-thriftserver -Dhadoop.version=2.2.0" "3030" &
```

Comment:
The names of the releases won't change, just their 'implementation'. This would have to select the Hadoop 1 profile and doesn't need to set version. Likewise for the next one.